solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -473,16 +456,13 @@ contract InterchainAccountRouter is
         bytes32 _router,
         bytes32 _ism
     ) private {
-        require(_router != bytes32(0), "invalid router address");
```

Comment:
why did you remove this check, I don't think you're checking for 0 addresses anywhere for isms or routers

Reply:
its already checked in Router